content/docs/how_we_work/open_source_policy.md

```diff
@@ -0,0 +1,30 @@
+---
+# Page title as it appears in the navigation menu
+title: "Open Source Policy"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 2
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Open Source Policy
+
+REG has a policy of publishing all the work we do under an open source licence, unless there are compelling reasons to make an exception.
+This typically means publishing software we write under one of the [OSI approved licences](https://opensource.org/licenses), or in the case of non-code contributions one of the [Creative Commons licences](https://creativecommons.org/share-your-work/).
```

Comment:
```suggestion
This typically means publishing software we write under one of the [OSI approved licences](https://opensource.org/licenses), or in the case of non-code contributions one of the [Creative Commons licences](https://creativecommons.org/share-your-work/) that supports derivative works.
```

The CC licences provide quite a scale and the [ND variants](https://creativecommons.org/licenses/by-nc-nd/4.0/) are quite restrictive and fall outside of the area I'd usually consider open source as it relates to code. Would it make sense to delineate?